replication/gatys_ecker_bethge_2016/main.py

```diff
@@ -51,10 +51,13 @@ def __init__(self, label, image, score_weight):
             impl_params=args.impl_params,
             hyper_parameters=hyper_parameters,
         )
-
+        filename = utils.make_output_filename(
+            ["gatys_ecker_betghe_2016", "fig_2", style_image.label],
```

Comment:
With above's suggestion you only need to remove the list:

```suggestion
            "gatys_ecker_betghe_2016", "fig_2", style_image.label,
```

Of course this needs to be applied to all calls.